CONTRIBUTING.md

```diff
@@ -16,13 +16,36 @@ Lastly, run the test command from the console:
 
     npm test
 
+
 ## Deploying
 
-- `npm run prepublish && npm run docs`
-- Update the version key in [package.json](https://github.com/mapbox/mapbox-gl-geocoder/blob/master/package.json#L3)
-- Outline changes in [CHANGELOG.md](https://github.com/mapbox/mapbox-gl-geocoder/blob/master/CHANGELOG.md)
-- Commit and push
-- `git tag -a vX.X.X -m 'vX.X.X'`
-- `git push --tags`
-- `npm publish`
-- Update version number in GL JS examples ([one](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder.html), [two](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/point-from-geocoder-result.html), [three](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-outside-the-map.html), [four](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-limit-region.html), [five](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-accept-coordinates.html), [six](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-proximity-bias.html), [seven](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/forward-geocode-custom-data.html))
+Follow this deploy process after all changes for the release are merged into master. You will copy and paste this checklist in the comment of the release pull request.
+
+```
+## Release checklist
+
+1. Create a branch off `master` and a pull request with the following changes. Copy this checklist in the comment of the pull request.
+    - [ ] Update the [CHANGELOG.md](https://github.com/mapbox/mapbox-gl-geocoder/blob/master/CHANGELOG.md) by comparing the last release and what is on `master`. In the changelog, replace the `master` heading with the to-be-released stable version.
+    - [ ] Update the version number in `package.json` and `package.lock.json`.
```

Comment:
nit picking on this but it should be `package-lock.json`. But if you update in package.json then npm install again, it will automatically update this.